pandas-stubs/_typing.pyi

```diff
@@ -44,9 +44,7 @@ PandasScalar: TypeAlias = Union[
 ]
 # Scalar: TypeAlias = Union[PythonScalar, PandasScalar]
 
-DatetimeLike: TypeAlias = Union[
-    datetime.date, datetime.datetime, np.datetime64, Timestamp
-]
+DatetimeLike: TypeAlias = Union[datetime.datetime, np.datetime64, Timestamp]
```

Comment:
It appears that `date` is incompatible with this change. `datetime` is a sub class of `date`, so the opposite change - removing `datetime`, would probably work.

Reply:
Explainer for the narrowing is in #369